akka-docs/rst/java/distributed-data.rst

```diff
@@ -503,6 +503,18 @@ Note that you should be prepared to receive ``WriteFailure`` as reply to an ``Up
 durable entry if the data could not be stored for some reason. When enabling ``write-behind-interval``
 such errors will only be logged and ``UpdateSuccess`` will still be the reply to the ``Update``.
 
+There is one important caveat when it comes pruning of :ref:`crdt_garbage_java` for durable data.
+If and old data entry that was never pruned is injected and merged with existing data after 
+that the pruning markers have been removed the value will not be correct. The time-to-live
+of the markers is defined by configuration 
+``akka.cluster.distributed-data.durable.remove-pruning-marker-after`` and is in the magnitude of days.
+This would be possible if replica with durable data didn't participate in the pruning
+(e.g. it was shutdown) and later started after this time. A durable replica should not 
+be stopped for longer time than this duration and if it is joining again after this
```

Comment:
I don't think we need to mention starting and stopping of the Replicator actor while the ActorSystem is running. That is not something that should be done, except for in akka tests. The Replicator is started when the ActorSystem is started and then left running until the ActorSystem is terminated.

I'm not sure how it can be clarified. Perhaps it was that I used the word `replica`? Removed that. See the update. Suggestions?